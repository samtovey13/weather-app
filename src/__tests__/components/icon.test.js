import React from "react";
import { render, waitFor } from "@testing-library/react";
import Icon from '../../components/icon';

describe("icon", () => {
  it("renders correctly", () => {
    const { asFragment } = render(<Icon icon="200" />)
    expect(asFragment()).toMatchSnapshot();
  })
})