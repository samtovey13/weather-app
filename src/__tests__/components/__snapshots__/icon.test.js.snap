// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`icon renders correctly 1`] = `
<DocumentFragment>
  <div>
    <i
      class="wi wi-day-storm-showers"
      name="owm"
    />
  </div>
</DocumentFragment>
`;
